src/pages/profile.js

```diff
@@ -55,7 +58,7 @@ export default () => {
           </div>  
           <div class="col-6">
             <ProfileText />
-            <Button variant="primary" size="lg">Sponsor now</Button>
```

Comment:
@SlinkEEE we sacrificed this button but think we should bring it back in a refactor ;)

Reply:
Ok we can add it back later <3